webpack.parts.js

```diff
@@ -190,8 +158,8 @@ exports.generateSourceMaps = ({ type }) => ({
 exports.newRelic = () => ({
     plugins: [
         new HtmlInjectNewRelicPlugin({
-            license: infraConfig.new_relic_client_license_key,
-            applicationID: infraConfig.new_relic_client_app_id,
+            license: process.env.NEW_RELIC_CLIENT_LICENSE_KEY,
+            applicationID: process.env.NEW_RELIC_CLIENT_APP_ID
```

Comment:
Are these baked into the client image? I am unclear where these should go/are needed.

Reply:
we'll clean this up later